packages/jsActions/native-mobile-resources/package.json

```diff
@@ -2,27 +2,32 @@
   "name": "native-mobile-resources",
   "version": "2.0.1",
   "license": "Apache-2.0",
+  "copyright": "Copyright © 2005-2021 Mendix Technology B.V. All rights reserved.",
   "repository": {
     "type": "git",
     "url": "https://github.com/mendix/widgets-resources.git"
   },
   "scripts": {
+    "prestart": "rimraf ./dist/tsc",
+    "start": "rollup --config ../../../configs/jsactions/rollup.config.js --watch --configProject nativemobileresources",
     "prebuild": "rimraf ./dist/tsc",
     "build": "rollup --config ../../../configs/jsactions/rollup.config.js --configProject nativemobileresources",
-    "version": "npm run build"
+    "version": "npm run build",
+    "postversion": "ts-node ../../../scripts/release/UpdateGithubRelease.ts --project ../../../scripts/tsconfig.json"
```

Comment:
Not needed, this script doesnt exist anymore

Reply:
Removed